src/index.css

```
@@ -154,11 +145,196 @@
+  .prose pre:has(code.language-sh),
+  .prose pre:has(code.language-shell),
+  .prose pre:has(code.language-zsh),
+  .prose pre:has(code.language-cmd),
+  .prose pre:has(code.language-bat),
+  .prose pre:has(code.language-powershell),
+  .prose pre:has(code.language-ps1) {
+    position: relative;
+  }
+
+  .prose pre code[class*="language-"] {
+    font-family: 'Cascadia Code', 'Fira Code', 'JetBrains Mono', 'SF Mono', 'Consolas', 'Liberation Mono', Menlo, monospace;
```

Stylelint (17.9.0)

[error] Expected no quotes around "Consolas" (font-family-name-quotes)

(font-family-name-quotes)